pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -570,6 +579,66 @@ func UnqualifiedSearchRegistries(ctx *types.SystemContext) ([]string, error) {
 	return config.UnqualifiedSearchRegistries, nil
 }
 
+// GetCredsStore return the list of credetntial store configed by creds_store for registries
+func GetCredsStore(ctx *types.SystemContext) ([]string, error) {
+	config, err := getConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return config.CredsStore, nil
+}
+
+// GetCredConfigs returns a slice of credential stores configred in registries.conf
+// caller should use GetRegistryCredentialHelpers to get cred helper of specified registry
+func GetCredConfigs(sys *types.SystemContext) ([]string, error) {
```

Comment:
```suggestion
// AllCredentialHelpers returns a list of all credential helpers that are configured for some registry.
// Almost all callers should use GetRegistryCredentialHelpers to get configuration applicable to a specific registry.
func AllCredentialHelpers(sys *types.SystemContext) ([]string, error) {
```
or something like that `AllConfiguredCredentialHelpers`?. `Get` is not idiomatic in Go.